synr/compiler.py

```diff
@@ -727,10 +727,10 @@ def my_function(x):
             else:
                 # make sure to preserve blank lines for correct spans
                 source = "\n".join([l[start_column:].rstrip() for l in lines])
-        mod = inspect.getmodule(program)
-        if mod is not None:
-            full_source = inspect.getsource(mod)
-        else:
+        try:
+            full_source, _ = inspect.findsource(program)
+            full_source = "".join(full_source)
+        except:
```

Comment:
Can you make this catch only the specific error type you expect?

Reply:
It seems like no exceptions here. I will delete it